infrastructure/azure-pipelines-dev.yml

```diff
@@ -8,38 +8,80 @@ resources:
       ref: refs/heads/main
 variables:
   phpVersion: "8.2"
-
+  serviceConnectionName: "Talent Cloud Service Connection"
+  npm_config_user_config: "$(System.DefaultWorkingDirectory)/.npmrc"
+  dirResults: "$(System.DefaultWorkingDirectory)/apps/playwright/test-results"
+  resourceGroupName: "rg_tbs_talent_2_PaaS"
+  vertical: "dev"
+  storageAccountName: "sttalclddevcacstdl"
+  storageContainer: "playwright-results"
+pool:
+  name: "TBS Linux Self Hosted"
 jobs:
-  - job: build_artifact
-    displayName: Build artifact
-    pool:
-      vmImage: ubuntu-22.04
+  - job: run_playwright_tests
+    displayName: Run Playwright tests
+    timeoutInMinutes: 30
```

Comment:
It seems like this is not long enough.  It's timing out for me, though it completes in around 25 minutes in GH.